netbox/utilities/mixins.py

```diff
@@ -0,0 +1,54 @@
+from django.db.models.query_utils import DeferredAttribute
+
+
+class Tracker(object):
+    def __init__(self, instance):
+        self.instance = instance
+        self.newly_created = False
+        self.changed = {}
+        self.tracked_fields = self.instance.change_tracking_fields
```

Comment:
This seems redundant. Won't the value of a Tracker instance's `tracked_fields` attribute always be equal to the value of `change_tracking_fields` on its instance?